tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Steps/UserSteps.cs

```diff
@@ -103,6 +103,37 @@ public async Task WhenAUserIsCreated(string organisationName, Table table)
             _response.Result = await client.PostAsync(new Uri($"{_organisationUrl}/{organisationId}/users"), content);
         }
 
+        [When(@"a GET request is made for a user with id (.*)")]
+        public async Task WhenAGETRequestIsMadeForAUserWithId(string userId)
+        {
+            using var client = new HttpClient();
+            client.SetBearerToken(_context.Get(ScenarioContextKeys.AccessToken, ""));
```

Comment:
Personally I prefer `string.Empty` because it leaves me in no doubt about intent but `""` is used elsewhere in this class so feel free to leave as is.

(There are currently no code standards around this either as far as I'm aware but I suspect it would be another developer's prerogative one anyway.)